### ISSUE TITLE
Username type validation

### ISSUE THREAD

**Issue by krishnakantnayak** [rewatiramanprasad/AutoSendEmail#1]
We can check username more efficiently by using endswith() function so that a username like Krishna.com@Kant
Won't get validated
We only want username like krishna@kant.com

**Comment by rewatiramanprasad** [comment]
yeah! you are right [krishnakant](https://github.com/DeathSage). This is one kind of other approach.
